src/main/java/org/beanpod/switchboard/service/UserService.java

```diff
@@ -0,0 +1,30 @@
+package org.beanpod.switchboard.service;
+
+import lombok.RequiredArgsConstructor;
+import org.beanpod.switchboard.dao.UserDaoImpl;
+import org.beanpod.switchboard.dto.UserDto;
+import org.openapitools.model.UserModel;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class UserService implements UserDetailsService {
+
+  private final UserDaoImpl userDao;
+  private final BCryptPasswordEncoder bCryptPasswordEncoder;
+
+  @Override
+  public UserDetails loadUserByUsername(String email) {
```

Comment:
this parameter name could also be changed to `username`

Reply:
ah i keep missing these